core/src/player.rs

```diff
@@ -477,22 +477,30 @@ impl Player {
 
             let max_frames_per_tick = self.max_frames_per_tick();
             let mut frame = 0;
+            let mut last_time_offset = self.time_offset;
```

Comment:
Shouldn't `self.time_offset` be always 0 outside the `while` loop below?

Reply:
In theory it could be changed outside the tick loop by things like ExternalInterface